### PR TITLE
Feat/eng calc method

### PR DESCRIPTION
**Pull request by corymosiman12**

### CODE REVIEW

**Review by markborkum — APPROVED** [review]
Changes look good, thanks.

**Review by macintoshpie — APPROVED** [review]
Looks good to me 👍

File: BuildingSync.xsd [hunk 1]
```diff
@@ -13861,7 +13867,16 @@
           </xs:sequence>
         </xs:complexType>
       </xs:element>
-      <xs:element name="Estimated" type="auc:EstimatedType" minOccurs="0"/>
+      <xs:element name="Estimated" type="auc:EstimatedType" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>The 'Estimated' calculation method is used to represent a scenario in which a guess or judgement call was used to derive data represented by this scenario type.</xs:documentation>
```
Comment (nllong) [inline BuildingSync.xsd:13872]: aka SWAG

File: proposals/2020/Add Engineering Calculation.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This proposal is to add the `auc:EngineeringCalculation` element as a calculation method.
+
+## Justification
+
+The `auc:CalculationMethodType` element is used as a child element of the `auc:ScenarioType` to help further refine how information in the scenario was derived.  This might include energy consumption information, costs, savings, etc.
+
+The current sequence of items that can be used to define an `auc:CalculationMethodType` are as follows:
+- `auc:Modeled` - used to represent when a whole building energy modeling application is used.
+- `auc:Measured` - used to represent when actual measurements (utility, AMI, HOBO devices, etc.) are used to capture real data from the building
+- `auc:Estimated` - used to represent a 'guess' or 'judegement call'
```
Comment (nllong) [inline proposals/2020/Add Engineering Calculation.md:14]:
```suggestion
- `auc:Estimated` - used to represent a 'guess' or 'judgment call'
```

**Review by nllong — APPROVED** [review]
Looks good.

**Pull request merged**